Code/Framework/AzCore/AzCore/Component/Component.h

```diff
@@ -238,7 +238,6 @@ namespace AZ
 
         Entity*     m_entity;       ///< Reference to the entity that owns the component. The value is null if the component is not attached to an entity.
         ComponentId m_id;           ///< A component ID that is unique for an entity. This component ID is not unique across all entities.
-        bool        m_isDeleting;   ///< Flag to prevent access to any member variables to the instance (possibly from other threads) while an instance is being deleted
```

Comment:
is it necessary to remove this field? based on comment, seems like it tries to protect it from multi-thread race condition.

Reply:
That was from a recent attempt to fix this, it no longer is valid (or needed)